dadi/lib/model/index.js

```diff
@@ -45,9 +45,9 @@ var Model = function (name, schema, conn, settings, database) {
 
     // add default handler to ensure there's no uncaught errors
     var self = this;
```

Comment:
I could be wrong, but I think we want to make sure there is always at least one `'error'` listener.  It probably depends on the versions we are using, but Node will blowup if `error` events aren't handled.

Maybe switch from `console.log` to our bunyan implementation?

Related: https://nodejs.org/docs/latest-v0.12.x/api/events.html#events_class_events_eventemitter